indico/web/client/styles/base/_theme.scss

```diff
@@ -12,9 +12,22 @@ $semantic-color-map: (
   warning: $yellow,
   visited: $purple,
   strong: $black,
+  rejected: $midnight-black,
+  make-changes: $bright-yellow,
+  request-changes: $dark-red,
 );
 
-$semantic-styles: (highlight, success, error, warning, visited, strong);
+$semantic-styles: (
```

Comment:
Can we avoid putting this in here? IIRC "semantic-styles" is pretty generic and meant for places where we have semantic names like "error" and "warning". But I don't think the editing-related ones belong here - they are specific to just that particular module...

If this is really needed because of the way the label is displayed, I suggest we use `editing-make-changes` etc and add a comment here telling people that it's not meant to be used elsewhere...